src/Microsoft.Health.Fhir.TemplateManagement/ArtifactProviders/IConvertDataTemplateProvider.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using DotLiquid;
+
+namespace Microsoft.Health.Fhir.TemplateManagement.ArtifactProviders
+{
+    public interface IConvertDataTemplateProvider
+    {
+        public Task<List<Dictionary<string, Template>>> GetTemplateCollectionAsync(CancellationToken cancellationToken);
```

Comment:
Could you clarify the intended returned object? 
It seems like the dictionary is (templateName, templateContents), so the list is to capture multiple sets of template collections?

Reply:
I think the list was intended to capture a base template collection and a user defined template collection. It may be useful in the future if we figure out a good way to overwrite the base settings with the users defined settings. For now we'll just assume that the user has uploaded all of the necessary templates to the storage account so there will only be one entry in the list.

The FHIR service defined a similar interface [here](https://github.com/microsoft/fhir-server/blob/10e30c9a8e977d674596d1dad731338a062029e2/src/Microsoft.Health.Fhir.Core/Features/Operations/ConvertData/IConvertDataTemplateProvider.cs#L14) which is useful but I didn't like that it accepted a ConvertData request as a parameter.